android/app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +33,12 @@
             <action android:name="android.intent.action.MAIN" />
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
+        <intent-filter android:label="filter_react_native">
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.DEFAULT" />
+            <category android:name="android.intent.category.BROWSABLE" />
+            <data android:scheme="catalog" android:host="scene" /> // A
```

Comment:
What does `// A` mean? Do we need it?

Reply:
What's the format of the final deeplink? `catalog://scene=3` ? `catalog://scene/3` ?

Reply:
`catalog://scene/3`, I added comment in `catalog_app.js`